content/en/docs/components/pipelines/user-guides/migration.md

```diff
@@ -1,258 +1,77 @@
-</tr>
-<tr>
-<td>
-
-```python
-from kfp.v2 import compiler
-# .json extension, deprecated format
-compiler.Compiler().compile(pipeline, package_path='my_pipeline.json')
-```
-
-</td>
-<td>
-
-```python
-from kfp import compiler
-# .yaml extension, preferred format
-compiler.Compiler().compile(pipeline, package_path='my_pipeline.yaml')
-```
-
-</td>
-</tr>
-</table>
+The key features introduced by KFP V2 are:
 
-### Breaking changes
+- A more pythonic SDK - use decorators like ([`@dsl.pipeline`][dsl-pipeline], [`@dsl.component`][dsl-component], [`@dsl.container_component`][dsl-container-component])
+- Decouple from Argo Workflows - compile pipelines to a generic [IR YAML][ir-yaml] rather than Argo `Workflow` YAML
+- Enhanced Workflow GUI - visualize pipelines, sub-DAGs (nested pipelines), loops, and artifacts (datasets, models, and metrics) to understand and debug your pipelines
 
-There are only a few subtle breaking changes in SDK v2 relative to the SDK v1 v2-namespace.
+## Version Matrix
 
-#### Drop support for Python 3.6
+The first version of [Kubeflow Platform](/docs/started/introduction/#what-is-kubeflow-platform) to include the Kubeflow Pipelines V2 backend was [Kubeflow 1.8](/docs/releases/kubeflow-1.8/).
 
-KFP SDK v1 supported Python 3.6. KFP SDK v2 supports Python >=3.7.0,\<3.12.0.
-
-#### CLI output change
-
-The v2 [KFP CLI][cli] is more consistent, readable, and parsable. Code that parsed the v1 CLI output may fail to parse the v2 CLI output.
-
-#### .after referencing upstream task in a dsl.ParallelFor loop
-
-The following pipeline cannot be compiled in KFP SDK v2:
-
-```python
-with dsl.ParallelFor(...):
-    t1 = comp()
-t2 = comp().after(t1)
-```
-
-This usage was primarily used by KFP SDK v1 users who implemented a custom `dsl.ParallelFor` fan-in. KFP SDK v2 natively supports fan-in from [`dsl.ParallelFor`][dsl-parallelfor] using [`dsl.Collected`][dsl-collected]. See [Control Flow][parallelfor-control-flow] user docs for instructions.
-
-#### Importer component import statement
-
-The location of the `importer_node` object has changed.
-
-**Change:** Import from `kfp.dsl`.
-
-<table>
-<tr>
-<th>Previous usage</th>
-<th>New usage</th>
-</tr>
-<tr>
-<td>
-
-```python
-from kfp.components import importer_node
-```
+The following table shows which versions of KFP backend are included in each version of Kubeflow Platform:
 
-</td>
-<td>
+Release Date | Kubeflow Platform Version | KFP Backend Version | SDK Mode: [`v1`](/docs/components/pipelines/legacy-v1/sdk/) | SDK Mode: [`v2`](/docs/components/pipelines/user-guides/core-functions/compile-a-pipeline/) | SDK Mode: [`v2-compatible`](https://v1-7-branch.kubeflow.org/docs/components/pipelines/v1/sdk-v2/)
+--- | --- | --- | --- | --- | ---
+Unreleased | [Kubeflow 1.9](/docs/releases/kubeflow-1.9/) | [2.2.0](https://github.com/kubeflow/pipelines/releases/tag/2.2.0) | <i class="fa-solid fa-check"></i> | <i class="fa-solid fa-check"></i> | <i class="fa-solid fa-xmark"></i>
```

Comment:
I think KFP 2.2.0 and 2.0.3 aren't v1 compatible, and it's not accurate with https://github.com/kubeflow/website/pull/3795/files#diff-8a058eaedaa59453e7bef895c3de4c7d5f0c736ff21c5c7120e6528761b6fb63R38

Reply:
Nevermind, I saw the table references the backend not SDK. This table looks good from BE perspective.